### ISSUE TITLE
qr_size not settable in vault_identity_mfa_totp resource

### ISSUE THREAD

**Issue by rkusler-intermedia** [hashicorp/terraform-provider-vault#1813]
### Terraform Version
Terraform v1.4.3-dev
on darwin_amd64
+ provider registry.terraform.io/hashicorp/vault v3.14.0

### Affected Resource(s)
Please list the resources as a list, for example:
- vault_identity_mfa_totp

### Terraform Configuration Files
```hcl
resource "vault_identity_mfa_totp" "mfa" {
  issuer    = "Vault"
  qr_size   = 200
}
```

### Debug Output
https://gist.github.com/rkusler-intermedia/a2813728338ebc4a115890fc70980a40

### Panic Output
n/a

### Expected Behavior
I would expect it to let me specify a QR code size using the qr_size attribute

### Actual Behavior
TF errors out saying that the qr_size attribute cannot be configured. If I do not configure a value then it gets set to zero which results in no QR codes being generated.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create a file file containing: 
```hcl
provider "vault" { }
resource "vault_identity_mfa_totp" "mfa" {
  issuer    = "Vault"
  qr_size   = 200
}
```
2. terraform init
3. terraform apply

### Important Factoids
I'm using the OSS version of Vault which requires that I use the 'MFA Login' version of MFA logins. The documentation for this module says that it is an option field with a default value of 200. However the documentation for Enterprise version of Vault (different vault system and different TF resource) does mention that the qr_size attribute is only used if the 'generate' attribute is set. The OSS 'MFA Login' documentation doesn't have a 'generate' attribute. 

Anyway I can manually modify the TOTP config by doing a 'vault write' and set the qr_size attribute and everything starts working. It would be nice if the TF provider would allow us to set this attribute using Terraform though.
```bash
vault write /identity/mfa/method/totp/86795013-ce2e-65e5-3482-28c860b47abf qa_size=200 issuer=Vault
```

### References
https://developer.hashicorp.com/vault/api-docs/secret/identity/mfa/totp (OSS MFA/TOTP)
https://developer.hashicorp.com/vault/tutorials/auth-methods/active-directory-mfa-login-totp (OSS walk-through)

https://developer.hashicorp.com/vault/api-docs/secret/totp (Enterprise MFA/TOTP included because I mentioned it above with respect to the 'generate' attribute)

**Comment by rkusler-intermedia** [comment]
Maybe it was fixed in https://github.com/hashicorp/terraform-provider-vault/commit/f86e54d09f341760a335c9ea623d77b4c6e7286f and this just hasn't made it's way into a release yet

**Comment by vinay-gopalan** [comment]
Hi, thanks for opening the issue and apologies for the delay. The [fix](https://github.com/hashicorp/terraform-provider-vault/pull/1750) for this was merged and has been released in [v3.15.0](https://github.com/hashicorp/terraform-provider-vault/releases/tag/v3.15.0). Going to close this issue, but please let us know if you face any other problems! 🙏🏼
